### PR TITLE
Added Mock to Test, and Guests/Posts to MockEventAccessor

### PR DESCRIPTION
**Pull request by NathanDeMaria**
Fixed the broken EventIndex test by adding Mock and Guests/Posts to the MockEventAccessor :smile:

### CODE REVIEW

File: Radabite.Tests/Controllers/EventControllerTest.cs [hunk 1]
```diff
@@ -47,7 +47,12 @@ public void EventIndex()
         {
             EventController controller = new EventController();
 
-            ViewResult result = controller.Index(123) as ViewResult;
+			var mock = new Mock<ControllerContext>();
+			mock.SetupGet(x => x.HttpContext.User.Identity.Name).Returns("Bob");
```
Comment (clottman) [inline Radabite.Tests/Controllers/EventControllerTest.cs:51]: You misspelled Bob.

**Comment by clottman** [comment]
:ship:

**Comment by clottman** [comment]
This is pretty neat. I'm glad we reviewed it.

File: Radabite.Tests/Controllers/EventControllerTest.cs (hunk 1 above)
Comment (sjager) [inline Radabite.Tests/Controllers/EventControllerTest.cs:51]: *Bawb

**Pull request merged**